client/src/app.js

```diff
@@ -0,0 +1,4 @@
+ReactDOM.render(
+    <h1>Hello, world!</h1>,
+    document.getElementById('root')
+  );
```

Comment:
We don't need to render again as we are already doing once in the `index.js` file. All we have to do is create a functional component and return the `h1` and then export the App component.

For example,

```suggestion
import React from 'react';

function App() {
  return (
    <div>
      <h1>Hello, World!</h1>
    </div>
  );
}

export default App;
```

Reply:
[https://github.com/Bong-Ninjas/weather-app/pull/9#discussion_r601352215](url)
You have told me earlier to render a hello world that's why i have done this.